yt_dlp/extractor/n1.py

```diff
@@ -140,7 +191,7 @@ def _real_extract(self, url):
                     'url': video_data.get('data-url'),
                     'id': video_data.get('id'),
                     'title': title,
-                    'thumbnail': video_data.get('data-thumbnail'),
+                    'thumbnail': video_data.get('data-thumbnail') or video_data.get('data-default_thumbnail'),
```

Comment:
```suggestion
                    'thumbnail': traverse_obj(video_data, (('data-thumbnail', 'data-default_thumbnail'), {url_or_none}, any)),
```
and change the imports to
```
from ..utils import extract_attributes, unified_timestamp, url_or_none
from ..utils.traversal import traverse_obj
```